src/modules/Home/constants/home.ts

```diff
@@ -0,0 +1,4 @@
+export const CHANGE_MAIN_SLIDE = 'HOME/CHANGE_MAIN_SLIDE';
+export const REQUEST_HOME = 'HOME/REQUEST_HOME';
+export const DISANIMATE_MAIN_SLIDER = 'HOME/DISANIMATE_MAIN_SLIDE';
```

Comment:
change name to be like -> STOP_MAIN_SLIDER_ANIMATION

Reply:
Done